ingestion/src/metadata/ingestion/source/metadata/amundsen.py

```diff
@@ -495,7 +503,31 @@ def get_type_primitive_type(self, data_type):
         return data_type
 
     def get_database_service(self, service_name: str) -> DatabaseService:
-        service = self.metadata.get_by_name(entity=DatabaseService, fqn=service_name)
+        """
+        Method to get and create Database Service
+        """
+        try:
+            file_path = (
+                Path(__file__).cwd() / "examples/amundsen_data/service_type.json"
```

Comment:
is this the best path to place the file? Looks like it is required, maybe we should ship it differently. We might be adding the examples directory in the CI docker build, but I am not sure we are passing this on the release docker image.

Reply:
also, if it is a static file. Instead of loading it should we just have a constant holding the contents in a dict or something?

Reply:
cc @ulixius9